x/protorev/keeper/epoch_hook_test.go

```diff
@@ -117,6 +121,77 @@ func (suite *KeeperTestSuite) TestEpochHook() {
 	suite.Require().Equal(totalNumberExpected, totalActuallySeen)
 }
 
+func (suite *KeeperTestSuite) TestGetHighestLiquidityPool() {
+	testCases := []struct {
+		name           string
+		baseDenomPools map[string]map[string]keeper.LiquidityPoolStruct
+		// NOTE: the test doesnot have any expect error because we donot check for error
+		// in the parameters that we pass
+	}{
+		{
+			name: "Get highest liquidity pools for all base denoms",
+			baseDenomPools: map[string]map[string]keeper.LiquidityPoolStruct{
+				"uosmo": {
+					"ethereum": {Liquidity: sdk.NewInt(10000000), PoolId: 1},
+					"usdc":     {Liquidity: sdk.NewInt(20000), PoolId: 1},
+				},
+				"atom": {
+					"ethereuma": {Liquidity: sdk.NewInt(300000), PoolId: 2},
+					"usdca":     {Liquidity: sdk.NewInt(400000), PoolId: 2},
+				},
```

Comment:
This doesn't make sense to me. Is uosmo the base denom, and these are refering to a usomo<>eth and a uosmo<>usdc pool? If so, why is the pool ID the same?

Reply:
simplified the tests now. It should make more sense